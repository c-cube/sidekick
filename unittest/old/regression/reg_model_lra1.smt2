
(set-logic QF_LRA)
(declare-const a Real)
(assert (= (* 3 a) 5))
(check-sat)
(get-model)
