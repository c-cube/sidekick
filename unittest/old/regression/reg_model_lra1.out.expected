(model
 (true := true) 
 (false := false) 
 (a := 5/3) 
 ((* 3 a) := 5) 
 (5 := 5) 
 ((= (* 3 a) 5) := true) 
 ((<= (* 3 a) 5) := true) 
 ((>= (* 3 a) 5) := true))
